scripts/docker-compose/imports/objects-api/fixtures/demodata.json

```diff
@@ -876,9 +876,9 @@
                 "pdf": "http://openzaak.local:8000/documenten/api/v1/enkelvoudiginformatieobjecten/37e16ddc-7992-418c-b5a0-54cf6680329e",
                 "csv" : "http://openzaak.local:8000/documenten/api/v1/enkelvoudiginformatieobjecten/aeaba696-4968-46a6-8b1e-016f503ed88d"
             },
-            "start_at": "2023-10-25",
+            "start_at": "1970-01-01",
```

Comment:
Why 1970? To have it as 0 millis?

Reply:
No, just to have an obviously fictional date. ;-) I think 2023 is a bit tricky in that respect so I choose for an obvious date when we were not yet working on ZAC. :-)